client/src/components/AccountLayout.js

```diff
@@ -1,38 +1,56 @@
 import React from 'react'
 import { Box, Heading, Main } from 'grommet'
 import { useRouter } from 'next/router'
+import { useNotifications } from 'hooks/useNotifications'
+import { NOTIFICATION_TYPES } from 'helpers/notificationTypes'
 import { useIsClient } from '../hooks/useIsClient'
 import { SideNav } from './SideNav'
 import Header from './Header'
 
+const getNotificationsForType = (notifications, type) => {
```

Comment:
you can set `notifications = []` so you dont have to bail...

If you set the default value then you can `return notifications.filter...` it will also be the empty array if you dont pass any to the function

Reply:
This function was only needed for sorting notifications by tab, so it got removed entirely.